vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java

```diff
@@ -56,7 +59,7 @@
 public class RichTextEditor
         extends GeneratedVaadinRichTextEditor<RichTextEditor, String>
         implements HasSize, HasValueChangeMode, InputNotifier, KeyNotifier,
-        CompositionNotifier {
+        CompositionNotifier, HasLabel {
```

Comment:
The `vaadin-rich-text-editor` web component does not support a label: https://github.com/vaadin/flow-components/issues/3691

Reply:
Thanks for your review @sissbruecker. I did this backport based on #2271 and that change is present in the original PR, but you are right about it not being supported. I have just removed it from this PR.

Should we remove this Interface from other, more up-to-date branches then? It was merged to both [master](https://github.com/vaadin/flow-components/blob/master/vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java) and [23.2](https://github.com/vaadin/flow-components/blob/23.2/vaadin-rich-text-editor-flow-parent/vaadin-rich-text-editor-flow/src/main/java/com/vaadin/flow/component/richtexteditor/RichTextEditor.java) branches.

Reply:
I have to check back with the team how we want to proceed with the versions that already contain the change, but removing it from this PR is definitely the right choice.